### ISSUE TITLE
Testimonials 

### ISSUE THREAD

**Issue by stpaultim** [backdrop-contrib/zeever#19]
- [ ] Remove link to Testimonials Page
- [ ] Add more testimonials (maybe?)

This is what it looks like on the original theme:
![image](https://github.com/backdrop-contrib/zeever/assets/3144571/a19a2902-6587-4d86-8339-69a57b9cf480)

For now, I'm just using the existing Testimonial recipe, which isn't quite what we need for this. But, maybe we make the current recipe work or at least add more sample content. Maybe, we should improve the original recipes with more sample content. 

![image](https://github.com/backdrop-contrib/zeever/assets/3144571/f67d5a08-b6cb-47bb-a3b9-f5412d3021c7)

**Comment by stpaultim** [comment]
Today we updated the Testimonials recipe to use some better profile pics. 

Also, we currently have Testimonials Page in the menu. This is not part of the original Zeever site and I don't think it's necessary. We list testimonials on front page and may add the block to some other pages, but there is no need for a page of its own.

![image](https://github.com/backdrop-contrib/zeever/assets/3144571/5c0531df-442b-4406-956c-7bf94f014506)

**Comment by stpaultim** [comment]
This link has been removed from the menu. I think we can close this for now.

**Comment by izmeez** [comment]
I agree that the testimonials may be improved by reducing the size of images, i.e. reducing the focus on who, and increasing the presence, size, prominence of what is being said.

**Comment by stpaultim** [comment]
@izmeez - Today I realized that I had already reduced the size of the testimonial images. But, my change was broken due to a syntax error in the css file. 

I've fixed that and am releasing the update in a few minutes. If you have other specific suggestions, feel free to post them here. 

This is a good example of why I am using a single issue queue. If one is testing the Zeever theme with the Digital Agency package, how would one know where to put this issue. This could be an issue for the Testimonial Recipe, it could be an issue for the Digital Agency Recipe Package, or it could be someething that is left to be fixed by the contributed theme Zeever. 

In fact, I have decided to address this issue in the Digital Agency Recipe, but I don't think that anyone else could have necessarily predicted this unless they are folling the project pretty closely. 

I'll reopen this issue for now, in case anyone has additional feedback on this feature.

**Comment by stpaultim** [comment]
@izmeez and others. This is the change. Should I do more?
Keeping in mind that individual contrib themes can do more than this.

BEFORE:

![image](https://github.com/user-attachments/assets/c0d13a1a-50b0-4253-8a98-7c026bf49878)

AFTER:

![image](https://github.com/user-attachments/assets/185a11e1-1935-4a7a-a5a8-30beddd73865)

**Comment by izmeez** [comment]
How about putting the text first maybe even with large leading graphic quote symbol, then have the image down with the name?

**Comment by stpaultim** [comment]
@izmeez 

Kinda like this?

![image](https://github.com/user-attachments/assets/6c207218-b2e0-403e-bcc4-f5a2f6d1a890)

:-)

If I have time before the 1st, I'll take a stab at this. I like it, it just feels like a bit more work and I'm lazy. 

The good news is that it will require (encourage) me to figure out how to use the new SVG icon API.

**Comment by izmeez** [comment]
Yes, much better. Although I'd suggest the quote be on the left of the actual text and be the correct opening bracket 66 to avoid it looking like a 99 centered all by itself. And maybe increase the text font size for a bit more prominence.

**Comment by stpaultim** [comment]
![image](https://github.com/user-attachments/assets/688c2e26-9fcc-43b5-b82f-980c9a509050)

@izmeez - After a lot of beating head against wall with icon api, I got this.

**Comment by stpaultim** [comment]
![image](https://github.com/user-attachments/assets/b2af6529-8bef-4872-bc37-3149578932ba)

**Comment by izmeez** [comment]
@stpaultim I like it. I would suggest using the opening quote: 
unicode `U+201C | LEFT DOUBLE QUOTATION MARK`

Do you want to add the sub-heading: "Feedback from our clients"?

**Comment by izmeez** [comment]
@stpaultim I did not realize you were trying to do this with an svg icon until I saw all the activity in the zulip chat and I see the phosphor svg icons in backdrop only have one icon for quotes, not a pair. I thought you were just going to do it with css, like https://stackoverflow.com/questions/32909991/adding-quotes-to-blockquote

**Comment by stpaultim** [comment]
@izmeez - I liked the idea of using this as an opporunity to try using the new Icon API in Backdrop. As you can see, it was my first time using it.

**Comment by izmeez** [comment]
There are some internet repos of svg quote icons that include both open quote and close quote icons:

https://www.flaticon.com/free-icons/quotes
https://www.svgrepo.com/vectors/Quote/
https://iconscout.com/icons/open-quote

and more ...

**Comment by stpaultim** [comment]
I took the advice from @laryn and simply used css to invert the single icon for dual purpose use. 

Digital Agency Recipe Package

![image](https://github.com/user-attachments/assets/d41b9715-0b14-4571-a296-4b04e8f3a9bd)

Zeever Theme

![image](https://github.com/user-attachments/assets/90319072-e327-4a20-ae9b-8a01b3448fa0)

> Do you want to add the sub-heading: "Feedback from our clients"?

I'm thinking I will leave the generic title in the actual "Testimonials" recipe. BUT, I can modify it to be a bit more opinionated in the Digital Agency Recipe Package.

**Comment by izmeez** [comment]
@stpaultim This is not quite right, the image appears to be mirrored not rotated. It should look like `66` and `99`. Also, it might look better if the icons are filled rather than open.

**Comment by stpaultim** [comment]
@izmeez - I've seen the quotes done both ways. 

(1) Rotated
(2) Flipped

![image](https://github.com/user-attachments/assets/f7e25584-4271-4aa7-85b4-4cf872967510)


After looking around a bit, it does seem like rotating them as you suggested is more common. So, I'll go with that (coming soon). 

My goal in the recipe package is make everything look good enough that it can be used out of the box, not to make a final polished theme. I'm assuming that for most sites, there will be a theme involved as well. 

I'd like to sticke with the quotes that are provided by Backdrop Core for the recipe. Each theme has the possibility to replace them with whatever quotes they want to use. Maybe we can demonstrate that in Zeever, but replacing them in Zeever with solid quotes.
